### PR TITLE
Adding a holistic view to Capillary, without polling at the moment.

### PR DESCRIPTION
**Pull request by jshirley**
This creates a table that joins the various endpoints through an AJAX call,
allowing async loading of all spout endpoints.

Fixes MID-110

### CODE REVIEW

File: app/assets/javascripts/topology.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        this.setState({ delta: this.DEFAULT_DELTA, error: true, message: r.responseText });
+      }
+    }.bind(this);
+    r.send()
+  },
+  componentWillMount: function() {
+    //this.refreshTimer = window.setInterval(this.refreshDelta, this.props.refreshInterval);
+    this.refreshDelta();
+  },
+  componentWillUnmount: function() {
```
Comment (gphat) [inline app/assets/javascripts/topology.js]: Nice of you to clean up! ;)

File: app/assets/javascripts/topology.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+  processDeltas: function(deltas) {
+    console.log('parse deltas', deltas);
+    var healthyPartitions = 0,
+        unhealthyPartitions = 0,
+        totalDelta = 0;
+
+
+    deltas.forEach(function(delta) {
+      var storm = delta.storm;
+      if ( storm === null ) {
```
Comment (gphat) [inline app/assets/javascripts/topology.js]: This is just "if there is a delta" as being healthy/unhealthy?

**Comment by gphat** [comment]
:+1: Looks good. I followed it pretty well and only had a few clarifying questions.

File: app/assets/javascripts/topology.js (hunk 1 above)
Comment (jshirley) [inline app/assets/javascripts/topology.js]: Eventually we will want polling and automatic updates here, but I wanted to make sure that it works first and we can talk about intervals.

File: app/assets/javascripts/topology.js (hunk 2 above)
Comment (jshirley) [inline app/assets/javascripts/topology.js]: Another point to discuss, should it be a percentage thing?

Right now I defined healthy as "storm === null", which means it isn't processing tuples at all. Just something to think about in the future, we don't expose the healthy/unhealthy (yet)

**Pull request merged**